TumeKyouen/View/StoneButton.m

```diff
@@ -31,7 +31,8 @@ - (id)initWithOptions:(int)size state:(int)defaultState {
             forControlEvents:UIControlEventTouchDown];
     }
 
-    return self;
+    return
```

Comment:
ERROR: Core Foundation/Objective-C
Returning 'self' while it is not set to the result of '[(super or self) init...]'